release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -298,6 +298,94 @@ With this release, the `MaxUnavailableStatefulSet` featureset configuration para
 [id="ocp-4-14-monitoring"]
 === Monitoring
 
+The monitoring stack for this release includes the following new and modified features.
+
+[id="ocp-4-14-monitoring-updates-to-monitoring-stack-components-and-dependencies"]
+==== Updates to monitoring stack components and dependencies
+This release includes the following version updates for monitoring stack components and dependencies:
+
+* kube-state-metrics to 2.9.2
+* node-exporter to 1.6.1
+* prom-label-proxy to 0.7.0
+* Prometheus to 2.46.0
+* prometheus-operator to 0.67.1
+
+[id="ocp-4-14-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+[NOTE]
+====
+Red{nbsp}Hat does not guarantee backward compatibility for recording rules or alerting rules.
+====
+
+* *New*
+** Added the `KubeDeploymentRolloutStuck` alert to monitor if the rollout of a deployment has not progressed for 15 minutes.
+** Added the `NodeSystemSaturation` alert to monitor resource saturation on a node.
+** Added the `NodeSystemdServiceFailed` alert to monitor the systemd service on a node.
+** Added the `NodeMemoryMajorPagesFaults` alert to monitor major page faults on a node.
+** Added the `PrometheusSDRefreshFailure` alert to monitor failed Prometheus service discoveries.
+
+* *Changed*
+** Modified the `KubeAggregatedAPIDown` alert and the `KubeAggregatedAPIErrors` alert to evaluate only metrics from the `apiserver` job.
+** Modified the `KubeCPUOvercommit` alert to evaluate only metrics from the `kube-state-metrics` job.
+** Modified the `NodeHighNumberConntrackEntriesUsed`, `NodeNetworkReceiveErrs` and `NodeNetworkTransmitErrs` alerts to evaluate only metrics from the `node-exporter` job.
+
+* *Removed*
+** Removed the `MultipleContainersOOMKilled` alert and replaced it with the `ContainersOOMKilled` alert, which uses the `container_runtime_crio_containers_oom_total` metric and notifies users if more than five containers on the host have been terminated by the out-of-memory (OOM) killer on the host.
```

Comment:
`MultipleContainersOOMKilled` alert is just removed and not replaced with `ContainersOOMKilled` alert, 
```
$ oc get prometheusrules --all-namespaces -oyaml | grep -i ContainersOOMKilled
no result
```
PR: https://github.com/openshift/cluster-monitoring-operator/pull/2071

Reply:
:+1:  yes, apologies if my earlier comment was confusing. The node team considered adding a `ContainersOOMKilled` but decided against it. https://github.com/openshift/machine-config-operator/pull/2666#discussion_r678023007 summarizes the issues with the alert. The goal is to alert on nodes being under memory pressure and we have separate alerts for that.
So maybe we should add something like "Removed the `MultipleContainersOOMKilled` alert for not being actionable. Nodes under memory pressure are covered by other alerts."

Reply:
Changed per Jan's suggestion